aperture.go

```diff
@@ -293,7 +312,51 @@ func (a *Aperture) Start(errChan chan error) error {
 		}, nil
 	}
 
-	if !a.cfg.Authenticator.Disable {
+	switch {
+	case a.cfg.LNC.Enable:
+		log.Infof("Using lnc's authenticator config")
+
+		authCfg := a.cfg.LNC
+
+		nodeConn := lnc.NewNodeConn(lncStore)
+		session, err := lnc.NewSession(
+			authCfg.Passphrase, authCfg.MailboxAddress,
+			authCfg.DevServer,
+		)
+		if err != nil {
+			return fmt.Errorf("unable to create lnc session: %w",
+				err)
+		}
+
+		err = nodeConn.OpenConn(session)
+		if err != nil {
+			return fmt.Errorf("unable to connect to lnd using "+
+				"lnc: %w", err)
+		}
+
+		client, err := nodeConn.Client()
+		if err != nil {
+			return err
+		}
+
+		a.challenger, err = NewLndChallenger(
+			client, genInvoiceReq, errChan,
+		)
+		if err != nil {
+			return err
+		}
+
+		err = a.challenger.Start()
+		if err != nil {
+			return fmt.Errorf("unable to start lnd challenger: %w",
+				err)
+		}
+
+		a.lncNodeConn = nodeConn
```

Comment:
I think what we actually need is the ability to tear down the challenger? So we don't need to hold the connection, and can go with the interface approach @ellemouton mentioned above.